src/printer.js

```diff
@@ -2342,6 +2354,7 @@ function typeIsFunction(type) {
 
 function printExportDeclaration(path, options, print) {
   const decl = path.getValue();
+  const semi =  options.noSemi ? "" : ";"
```

Comment:
nit: two spaces, and missing semi :p But feel free to leave it as is, it'll go away when we reformat it again :)